USERGUIDE.md

```diff
@@ -58,11 +58,7 @@ measure_ts,temperature
 
 First, setup the datalake by creating a table in the postgresql database. 
 
-Head to the pgadmin interface and execute the [table creation script](examples/basic/create_datalake_tables.sql).
-
-```
-minikube service fadi-pgadmin -n fadi
-```
+Head to the pgadmin interface (`fadi.minikube/pgadmin`) and execute the [table creation script](examples/basic/create_datalake_tables.sql).
```

Comment:
This should be a link: `fadi.minikube/pgadmin` --> http://fadi.minikube/pgadmin, same for the others below